.github/workflows/cd_dev.yaml

```diff
@@ -93,31 +89,6 @@ jobs:
           gh auth login --with-token < <(echo ${{ secrets.GITHUB_TOKEN }})
           curl -fsS https://packages.fluvio.io/v1/install.sh | bash
 
-      # Download artifacts
-      - name: Download artifact - fluvio
-        uses: actions/download-artifact@v2
-        with:
-          name: fluvio-x86_64-unknown-linux-musl
-          path: .
-      - name: Download artifact - flv-test
-        uses: actions/download-artifact@v2
-        with:
-          name: flv-test-x86_64-unknown-linux-musl
-          path: .
-      - name: Download Docker Image as Artifact
-        uses: actions/download-artifact@v2
-        with:
-          name: infinyon-fluvio
-          path: /tmp
-      - name: Load Fluvio Docker Image
-        run: |
-          ls -la /tmp
-          docker image load --input /tmp/infinyon-fluvio.tar
-          docker image ls -a
-
-      - name: Print artifacts and mark executable
-        run: ls -la . && chmod +x ./fluvio ./flv-test && ./fluvio -h && ./flv-test -h
```

Comment:
During CI, the upgrade script assumes the Fluvio binary is located at `$(pwd)/fluvio` and docker image loaded in order to do a cluster upgrade.